### PR TITLE
Backported "--no-proxy" fix from SLE-11-SP4 branch

### PR DESCRIPTION
**Pull request by mchf**
bsc#923788

https://github.com/yast/yast-network/pull/293

### CODE REVIEW

File: src/modules/Proxy.ycp [hunk 1]
```diff
@@ -99,6 +99,68 @@ global define boolean Read() {
...
+    if (user != "") {
+        proxyuser = user;
+        if (pass != "") proxyuser = user + ":" + pass;
+    }
+
+    map<string, string> options = $[
+        "--proxy-user": proxyuser,
+        // bnc#305163
+        "--proxy":      http,
+        // bsc#923788
```
Comment (jreidinger) [inline src/modules/Proxy.ycp]: please add to bnc at least brief overview what to expect there, a summary

**Comment by jreidinger** [comment]
in general looks good, just comment with bnc only is not so nice.

**Comment by jreidinger** [comment]
LGTM

**Comment by mvidner** [comment]
LGTM, thanks!

**Comment by mchf** [comment]
Thanks for reviews.

**Pull request merged**